cmd/boulder-publisher/main.go

```diff
@@ -83,6 +84,20 @@ func main() {
 	gw := bgrpc.NewPublisherServerWrapper(pubi)
 	pubPB.RegisterPublisherServer(grpcSrv, gw)
 
+	// Collect HTTP GET debug data every 30 seconds from each log which
+	// we are requesting SCTs from. This will allow us to verify during
+	// CT outages we've seen in the past if the issue is with the CT
+	// client itself or something in the larger publisher/golang http
+	// library.
+	if features.Enabled(features.ProbeCTLogs) {
+		go func() {
+			t := time.NewTicker(time.Second * 30)
```

Comment:
I think we should do this once per second, to get more fine-grained data, since the outages can be pretty short. Also, since `probeLogs` waits for all results to return, we should put the call to `probeLogs` in a goroutine so we're consistently firing off one thing per time window.